tests/e2e-tests/specs/backend/reviews-by-category.test.js

```diff
@@ -32,7 +40,11 @@ describe( `${ block.name } Block`, () => {
 		await page.waitForSelector(
 			`${ block.class } .woocommerce-search-list__item`
 		);
-		await page.click( `${ block.class } .woocommerce-search-list__item` );
+		const categoryWithReviews = await findElementWithText(
+			`.woocommerce-search-list__item`,
+			fixtureProducts()[ 0 ].categories[ 0 ]
+		);
+		await categoryWithReviews.click();
```

Comment:
I don't know why this test is failing for me here. I tried debugging it with `npm run test:e2e-dev` and I saw only the 'Uncategorized' item is shown in the selector:

![imatge](https://user-images.githubusercontent.com/3616980/91400057-57181080-e83f-11ea-969d-df92a231f51a.png)

I can keep investigating what's going on, but if somebody has any idea, that would be appreciated. :slightly_smiling_face:

Reply:
It seems you don't have a category named `Music` that should be automatically created as part of the fixtures, I'm not really sure why your instance fails to insert it.
https://github.com/woocommerce/woocommerce-gutenberg-products-block//blob/116e962aa4b1cd63252a18988506d911a5da2814/tests/e2e-tests/config/setup.js#L36